google-cloud-security_center/lib/google/cloud/security_center/v1/security_center_client.rb

```diff
@@ -0,0 +1,1772 @@
+            "organizations/{organization}/sources/{source}/findings/{finding}/securityMarks"
+          )
+
+          private_constant :FINDING_SECURITY_MARKS_PATH_TEMPLATE
+
+          ORGANIZATION_PATH_TEMPLATE = Google::Gax::PathTemplate.new(
+            "organizations/{organization}"
+          )
+
+          private_constant :ORGANIZATION_PATH_TEMPLATE
+
+          ORGANIZATION_SETTINGS_PATH_TEMPLATE = Google::Gax::PathTemplate.new(
+            "organizations/{organization}/organizationSettings"
+          )
+
+          private_constant :ORGANIZATION_SETTINGS_PATH_TEMPLATE
+
+          ORGANIZATION_SOURCES_PATH_TEMPLATE = Google::Gax::PathTemplate.new(
+            "organizations/{organization}/sources/-"
+          )
+
+          private_constant :ORGANIZATION_SOURCES_PATH_TEMPLATE
+
+          SOURCE_PATH_TEMPLATE = Google::Gax::PathTemplate.new(
+            "organizations/{organization}/sources/{source}"
+          )
+
+          private_constant :SOURCE_PATH_TEMPLATE
+
+          # Returns a fully-qualified asset resource name string.
+          # @param organization [String]
+          # @param asset [String]
+          # @return [String]
+          def self.asset_path organization, asset
+            ASSET_PATH_TEMPLATE.render(
+              :"organization" => organization,
+              :"asset" => asset
+            )
+          end
+
+          # Returns a fully-qualified asset_security_marks resource name string.
+          # @param organization [String]
+          # @param asset [String]
+          # @return [String]
+          def self.asset_security_marks_path organization, asset
+            ASSET_SECURITY_MARKS_PATH_TEMPLATE.render(
+              :"organization" => organization,
+              :"asset" => asset
+            )
+          end
+
+          # Returns a fully-qualified finding resource name string.
+          # @param organization [String]
+          # @param source [String]
+          # @param finding [String]
+          # @return [String]
+          def self.finding_path organization, source, finding
+            FINDING_PATH_TEMPLATE.render(
+              :"organization" => organization,
+              :"source" => source,
+              :"finding" => finding
+            )
+          end
+
+          # Returns a fully-qualified finding_security_marks resource name string.
+          # @param organization [String]
+          # @param source [String]
+          # @param finding [String]
+          # @return [String]
+          def self.finding_security_marks_path organization, source, finding
+            FINDING_SECURITY_MARKS_PATH_TEMPLATE.render(
+              :"organization" => organization,
+              :"source" => source,
+              :"finding" => finding
+            )
+          end
+
+          # Returns a fully-qualified organization resource name string.
+          # @param organization [String]
+          # @return [String]
+          def self.organization_path organization
```

Comment:
should this have the instance method aliases like https://github.com/googleapis/google-cloud-ruby/pull/3127 ?

Reply:
Added